numpy/ma/core.py

```diff
@@ -3509,7 +3517,10 @@ def soften_mask(self):
         return self
 
     hardmask = property(fget=lambda self: self._hardmask,
-                        doc="Hardness of the mask")
+                        doc="""
+        Returns whether the mask is hard (``True``) or soft (``False``).
+        When the mask is hard, masked entries cannot be unmasked.
+        """)
```

Comment:
nit: Could switch this one and a few others to `@property` syntax, if you're feeling generour

Reply:
Alternatively, an easy starter issue: #13024

Reply:
#13025

Reply:
whoops, duplicate issue. Closed #13025